docs/topics/testing/advanced.txt

```diff
@@ -568,6 +568,13 @@ Methods
 
     Returns the result produced by the running the test suite.
 
+.. method:: DiscoverRunner.get_test_runner_kwargs()
+
+    Returns the ``**kwargs`` to use to instantiate the
+    ``DiscoverRunner.test_runner``.
+
+    Override this method to customize these ``**kwargs``.
```

Comment:
I think this could be removed. Obviously(?) all methods can be overridden to customize them.